src/els_dt_documents.erl

```diff
@@ -0,0 +1,83 @@
+%%==============================================================================
+%% The 'documents' table
+%%==============================================================================
+
+-module(els_dt_documents).
```

Comment:
What does `dt` stand for here?

Reply:
Db Table. It's the name of the behaviour. I'm open for suggestion.

Reply:
I don't have any better suggestions to be honest 😜